ngraph/core/include/ngraph/op/space_to_batch.hpp

```diff
@@ -60,6 +60,9 @@ namespace ngraph
                 std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
                 bool visit_attributes(AttributeVisitor& visitor) override;
+
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
On the implementation moment, there was a decision (discuessed with @ilyachur  and @lazarevevgeny ) that non-template references can be added directly to evaluate. 
Are there any changes since that time?

Reply:
No it is correct. I just asked in order to avoid evaluate methods with template code